tests/rptest/tests/schema_registry_test.py

```diff
@@ -1112,3 +1112,39 @@ def test_serde_client(self):
                 assert schema.json().get("schemaType") is None
             else:
                 assert schema.json()["schemaType"] == protocols[i].name
+
+
+class SchemaRegistryBasicAuthTest(SchemaRegistryEndpoints):
+    """
+    Test schema registry against a redpanda cluster with HTTP Basic Auth enabled.
+    """
+    username = 'red'
+    password = 'panda'
+
+    def __init__(self, context):
+        security = SecurityConfig()
+        security.enable_sasl = True
+        security.endpoint_authn_method = 'sasl'
+        security.sr_authn_method = 'http_basic'
+
+        super(SchemaRegistryBasicAuthTest, self).__init__(context,
+                                                          security=security)
+
+    @cluster(num_nodes=3)
+    def test_schemas_types(self):
+        """
+        Verify the schema registry returns the supported types
+        """
+        result_raw = self._get_schemas_types(headers={},
```

Comment:
I think the empty `headers` adds confusion.

Reply:
>I think the empty headers adds confusion.
+1